unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarModifiers/TriggerArea.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+[System.Serializable]
+public abstract class TriggerArea
+{
+    public abstract void AddCollider(GameObject avatarModifierArea);
+    public abstract Collider GetCollider(GameObject avatarModifierArea);
+
+}
+
+[System.Serializable]
+public class BoxTriggerArea : TriggerArea
+{
+    public Vector3 box;
+
+    public override void AddCollider(GameObject avatarModifierArea)
+    {
+        BoxCollider boxCollider = avatarModifierArea.AddComponent<BoxCollider>();
+        boxCollider.isTrigger = true;
+        boxCollider.size = box;
+    }
+
+    public override Collider GetCollider(GameObject avatarModifierArea)
+    {
+        return avatarModifierArea.GetComponent<BoxCollider>();
```

Comment:
We should caché the `boxCollider` in the class (at `AddCollider`), as every GetComponent call is costly and should be avoided whenever we can